modules/tracker/mcserver.py

```diff
@@ -85,9 +85,9 @@ def get_favicon_path(self):
 
     def get_motd_path(self):
         if self.status:
-            data = "http://status.mclive.eu/{}/{}/25565/banner.png".format(self.get_name(), self.server_address)
+            data = "https://api.loohpjames.com/serverbanner.png?ip={}&name={}".format(self.server_address(), self.get_name)
```

Comment:
There's a possible error occuring in this line
self.server_address() should be changed to self.server_address
and self.get_name should be self.get_name()
please fix these problems and do a fresh commit

Reply:
Fixed !